test/e2e/multi-cluster-deployment/run-test.sh

```diff
@@ -18,25 +18,43 @@
 
 set -x # so users can see what is going on
 
+env="kind"
+
 if [ "$1" == "--released" ]; then
     setup_flags="$1"
     shift
 fi
 
-if [ "$#" != 0 ]; then
-    echo "Usage: $0 [--released]" >& 2
-    exit 1
+if [ "$1" == "--env" ]; then
+    env="$2"
+    shift
 fi
 
+# if [ "$#" != 0 ]; then
+#     echo "Usage: $0 [--released]" >& 2
+#     exit 1
+# fi
+
 set -e # exit on error
 
-SRC_DIR="$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)"
-COMMON_SRCS="${SRC_DIR}/../common"
-HACK_DIR="${SRC_DIR}/../../../hack"
+if [ $env == "kind" ];then
+    SRC_DIR="$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)"
+    COMMON_SRCS="${SRC_DIR}/../common"
+    HACK_DIR="${SRC_DIR}/../../../hack"
+
+    "${HACK_DIR}/check_pre_req.sh" --assert --verbose kubectl docker kind make go ko yq helm kflex ocm
 
-"${HACK_DIR}/check_pre_req.sh" --assert --verbose kubectl docker kind make go ko yq helm kflex ocm
+    "${COMMON_SRCS}/cleanup.sh"
+    source "${COMMON_SRCS}/setup-shell.sh"
+    "${COMMON_SRCS}/setup-kubestellar.sh" $setup_flags
+    "${SRC_DIR}/use-kubestellar.sh"
+fi
+
+
+if [ $env == "ocp" ];then
+   bash <(curl -s https://raw.githubusercontent.com/kubestellar/kubestellar/release-$KUBESTELLAR_VERSION/test/e2e/common/cleanup.sh) --env ocp
+   source <(curl -s https://raw.githubusercontent.com/kubestellar/kubestellar/release-$KUBESTELLAR_VERSION/test/e2e/common/setup-shell.sh)
+   bash <(curl -s https://raw.githubusercontent.com/kubestellar/kubestellar/release-$KUBESTELLAR_VERSION/test/e2e/common/setup-kubestellar-ocp.sh)
+   bash <(curl -s https://raw.githubusercontent.com/kubestellar/kubestellar/release-$KUBESTELLAR_VERSION/test/e2e/multi-cluster-deployment/use-kubestellar.sh) --env ocp
+fi
 
```

Comment:
What should happen if `$env` is neither "kind" nor "ocp"? I think that a silent nothing is not a good answer. I think that an error message would be better.

Reply:
Fixed - added an error message